data/datamgr.py

```diff
@@ -81,3 +81,20 @@ def get_data_loader(self, data_file, aug): #parameters that would change on trai
         return data_loader
 
 
+class SetDataManager_small(DataManager):
```

Comment:
I don't see a change except `n_eposide =20`. Is that the only intended change?
In that case let `SetDataManager_small` be a subclass of `SetDataManager` and just call the constructor of `SetDataManager` in the constructor of `SetDataManager_small` with the lower number of episodes. 

Other question: Do we really need an extra class for it?